### PR TITLE
remove `colorScheme` and `variant` from Button props

### PR DESCRIPTION
**Pull request by jaf107**
resolves : https://github.com/optimizely-axiom/optiaxiom/issues/265

### CODE REVIEW

File: packages/react/src/toast-action/ToastAction.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const ToastAction = forwardRef<HTMLButtonElement, ToastProps>(
           orientation="vertical"
         />
 
-        <Button asChild size="sm" variant="solid" {...props}>
+        <Button appearance="secondary" asChild size="sm" {...props}>
```
Comment (radnan) [inline packages/react/src/toast-action/ToastAction.tsx:21]: let's add a new appearance for these called `inverse`

so `inverse` will use `colorScheme="neutral"` and `variant="solid"`

File: packages/react/src/toast/Toast.tsx [hunk 2]
```diff
@@ -46,10 +46,10 @@ export const Toast = forwardRef<HTMLLIElement, ToastProps>(
 
           <RadixToast.Close asChild>
             <Button
+              appearance="secondary"
```
Comment (radnan) [inline packages/react/src/toast/Toast.tsx:49]: same comment - use the new `inverse` appearance here

File: packages/react/src/toast-action/ToastAction.tsx (hunk 1 above)
Comment (jaf107) [reply packages/react/src/toast-action/ToastAction.tsx:21]: should we show it in storybook bhai?

Comment (radnan) [reply packages/react/src/toast-action/ToastAction.tsx]: Nah we don’t need to show it there - the toast stories will cover them

**Review by radnan — APPROVED** [review]

**Pull request merged**